scripts/setup.sh

```diff
@@ -3,5 +3,12 @@
 set -ex
 
 cd /external
-rm -rf node_modules
+
+for arg in "$@"; do
+  if [ "$arg" = "--clean" ]; then
+    echo "Removing node_modules..."
+    rm -rf node_modules/
```

Comment:
I‘d suggest the following:

Instead of using rm rf we can use `npm ci` which does a clean install for the `—clean` case.

Otherwise let’s use regular `npm i`

Reply:
You don't need to trust npm is doing the right thing and cleaning up properly if you `rm -rf`.

Reply:
Fair enough, then let’s just remove the npm ci command, otherwise we won’t gain much 👍